hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/qjournal/server/JournalNodeSyncer.java

```diff
@@ -174,8 +174,8 @@ private void startSyncJournalsDaemon() {
         }
       }
       if (!createEditsSyncDir()) {
-        LOG.error("Failed to create directory for downloading log " +
-                "segments: %s. Stopping Journal Node Sync.",
+        LOG.error("Failed to create directory for downloading log "
+                + "segments: {}. Stopping Journal Node Sync.",
```

Comment:
Guess CI will report checkstyle problem.

Reply:
Sure, let me put it back to how it was.